python/packages/nisar/workflows/gcov_runconfig.py

```diff
@@ -16,6 +18,94 @@ def load(self):
         '''
         Load GCOV specific parameters.
         '''
+        warning_channel = journal.warning('GCOVRunConfig.load()')
+
+        flag_fullcovariance = self.cfg['processing']['input_subset'][
+            'fullcovariance']
+
+        # Handle the case in which the `fullcovariance` flag field is empty.
+        # If so, the YAML parser assigns it the string value `"None"`.
+        if (flag_fullcovariance is None or
+                (isinstance(flag_fullcovariance, str) and
+                 flag_fullcovariance == 'None')):
```

Comment:
I think we shouldn't need this if you adopt the previous comment

```suggestion
        if flag_fullcovariance is None:
```